packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -46,13 +46,18 @@ function getImgNodeKey(elementId) {
   return `${IMG_NODE}_${elementId}`;
 }
 
-export function getMediaBaseColor(resource, onBaseColor) {
+export function getResourceBaseColor(resource) {
   const { type, src, poster } = resource;
-  setOrCreateImage({
-    src: type === 'video' ? poster : src,
+
+  return setOrCreateImage({
+    src: type === 'image' ? src : poster,
     width: 10,
     height: 'auto',
-  }).then(
+  });
+}
+
+export function getMediaBaseColor(resource, onBaseColor) {
```

Comment:
If I am not mistaken, this function is now completely unused and could be removed

Reply:
It still used here. https://github.com/google/web-stories-wp/blob/a6509894b5da58af92823b607c05751f6c3822a0/packages/story-editor/src/components/panels/design/sizePosition/sizePosition.js#L240

Can we remove the base color out of size position @miina ?